From 95f303e3bdce850f0bddcfde826db0485837f104 Mon Sep 17 00:00:00 2001
From: Tao Ren <taoren@meta.com>
Date: Thu, 15 Jun 2023 15:51:09 -0700
Subject: [PATCH 1003/1003] ARM: dts: aspeed: yamp: Enable jtag controller

Enable jtag controller for FPGA firmware upgrade. Refer to "fpga_util.sh"
in meta-yamp layer for details.
---
 arch/arm/boot/dts/aspeed-bmc-facebook-yamp.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed-bmc-facebook-yamp.dts b/arch/arm/boot/dts/aspeed-bmc-facebook-yamp.dts
index f4c6efe29b80..e38b56cb9072 100644
--- a/arch/arm/boot/dts/aspeed-bmc-facebook-yamp.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-facebook-yamp.dts
@@ -144,3 +144,7 @@ flash1@0 {
 		};
 	};
 };
+
+&jtag {
+	status = "okay";
+};
-- 
2.34.1

